src/main/resources/application.yml

```diff
@@ -120,28 +136,28 @@ initializr:
           mappings:
           - versionRange: "[1.0.0.RELEASE,2.0.0.RELEASE)"
             version: 1.3.1.RELEASE
-          - versionRange: "[2.0.0.RELEASE,3.0.0.RELEASE)"
-            version: 2.0.0.RELEASE
-            repositories: spring-snapshots,spring-milestones
+          - versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"
```

Comment:
We dropped counter in the 2.1.0.RELEASE.

Reply:
`- versionRange: "[2.0.0.RELEASE,2.1.0.RELEASE)"` means excluding `2.1.0.RELEASE`